### PR TITLE
Use node-notifier instead of growl for notifications

### PR DESCRIPTION
**Pull request by k88hudson**
What do you think of using node-notifier instead of growl? It's cross-platform compatible and falls back to growl. 

I also added a couple of cli options, let me know what you think.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "growl": "1.7.0",
```
Comment (jgonera) [inline package.json]: Should this be removed?

File: bin/autoless [hunk 2]
```diff
@@ -13,6 +13,8 @@ program
   .usage('[options] <source_dir> [destination_dir]')
   .option('--interval <ms>', 'How often files are checked for changes', 100)
   .option('--no-watch', "Compile what needs to be compiled and exit")
+  .option('--no-notify', "Do not send any notifications")
```
Comment (jgonera) [inline bin/autoless]: Can you also add these to the readme?

**Comment by k88hudson** [comment]
@jgonera oops, sorry about that. requested fixes are in!

**Comment by jgonera** [comment]
Sorry for the delay. Thanks!

**Pull request merged**